elastic/kibana
# [Infra] Remove container and host tsvb fields

Closes #197993
## Summary

This PR removes the old tsvb fields used for containers before we changed to the asset details view. They were used for displaying charts but now we use lens formulas so those are no longer needed

## Testing

This PR removes old no longer used code so the testing should be for regressions
- Go to Infrastructure Inventory
- Select Docker Container from the `Show` drop-down
- Click on any container: 
![image](https://github.com/user-attachments/assets/20f66745-4cbd-4700-b2c5-c43ee753b2cb)

- The asset details flyout and full page view should load as before (the same as for hosts)
- Repeat same steps for host (and k8s pod)
